misc/windows-deploy/user-data.ps1

```diff
@@ -0,0 +1,52 @@
+<powershell>
+## The string 'windows' shoud be replaced with 
+## template variable for cluster name.
+
+# Set agent env variables for the Machine context (durrable)
+[Environment]::SetEnvironmentVariable("ECS_CLUSTER", "windows", "Machine")
+[Environment]::SetEnvironmentVariable("ECS_CRED_PROXY_ENABLED", "False", "Machine")
```

Comment:
Can you change this to `ECS_ENABLE_TASK_IAM_ROLE` and use `false` instead of `False`?